whitepaper.md

```diff
@@ -458,7 +458,7 @@ Let us start this section with a word of warning: You will find correlations in
 
 While threshold-based alerting provides users with a mechanism to configure alerts based on known values, they can be rigid and unable to adapt to variations in the data caused by seasonality, ongoing rollouts and other scenarios.
 
-The usage of machine learning techniques and statistical models helps understand several months' worth of behavioural patterns and use that to determine if the current sample that is observed is anomalous or not. Several scholarly works and open-source implementations exist to adopt anomaly detection as a mechanism to detect issues in the system that is being observed. See the [eBay user story](https://tech.ebayinc.com/engineering/sherlock.io-an-upgraded-machine-learning-monitoring-system) on this subject.
+The usage of machine learning techniques and statistical models helps understand several months' worth of behavioural patterns and use that to determine if the current sample that is observed is anomalous or not. Several scholarly works and open-source implementations exist to adopt anomaly detection as a mechanism to detect issues in the system that is being observed. See the [eBay user story](https://ebayinc.to/4840wUR) on this subject.
```

Comment:
Good point. The short link you provided @wbollock resolves to https://innovation.ebayinc.com/tech/engineering/sherlock.io-an-upgraded-machine-learning-monitoring-system/ - perhaps better use this as it's more descriptive?
```suggestion
The usage of machine learning techniques and statistical models helps understand several months' worth of behavioural patterns and use that to determine if the current sample that is observed is anomalous or not. Several scholarly works and open-source implementations exist to adopt anomaly detection as a mechanism to detect issues in the system that is being observed. See the [eBay user story](https://innovation.ebayinc.com/tech/engineering/sherlock.io-an-upgraded-machine-learning-monitoring-system/) on this subject.
```

Reply:
thanks! this works better